content/en/platform/corda/5.0-beta/_index.md

```diff
@@ -13,7 +13,7 @@ menu:
     name: Corda 5.0 Beta 2
 ---
 
-## Corda 5.0 Beta 2
+## Corda 5.0: Next-Gen Corda - Beta 2
```

Comment:
This seems a little clumsy to me, almost as though we're naming something twice. I think it should be "Next-Gen Corda - Beta 2"